app.py

```diff
@@ -61,36 +64,8 @@ def graphcall():
         ).json()
     return render_template('display.html', result=graph_data)
 
-
-def _load_cache():
-    cache = msal.SerializableTokenCache()
-    if session.get("token_cache"):
-        cache.deserialize(session["token_cache"])
-    return cache
-
-def _save_cache(cache):
-    if cache.has_state_changed:
-        session["token_cache"] = cache.serialize()
-
-def _build_msal_app(cache=None, authority=None):
-    return msal.ConfidentialClientApplication(
-        app_config.CLIENT_ID, authority=authority or app_config.AUTHORITY,
-        client_credential=app_config.CLIENT_SECRET, token_cache=cache)
-
 def _build_auth_url(authority=None, scopes=None, state=None):
-    return _build_msal_app(authority=authority).get_authorization_request_url(
-        scopes or [],
-        state=state or str(uuid.uuid4()),
-        redirect_uri=url_for("authorized", _external=True))
-
-def _get_token_from_cache(scope=None):
```

Comment:
@rayluo this is the `_save_cache(cache)` i was referring to in our [other review conversation](https://github.com/Azure-Samples/ms-identity-python-webapp/pull/15#discussion_r373817008).